### PR TITLE
Browser sync support

### PR DESCRIPTION
**Pull request by ojacquemart**
Hi,

This is a basic browser-sync support for development.

Great work, btw :+1:

### CODE REVIEW

**Comment by ojacquemart** [comment]
Maybe you should not consider this PR. It seems to break the hot-loader.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ It is something similar to the Angular Quick Start but does the entire build wit
 - Sample unit tests with Jasmine and Karma.
 - End-to-end tests with Protractor.
 - Development server with Livereload.
+- Time-saving synchronised browser testing with Browsersync.
```
Comment (mgechev) [inline README.md]: Lets announce that after the A2 support by browsersync is ready.

**Comment by mgechev** [comment]
@ojacquemart great job with the integration! I played a bit with your PR and here is how I fixed the [hot loading](https://gist.github.com/mgechev/ac7392be88865698547e) support.

**Comment by ojacquemart** [comment]
Sweet, I was on the right way to fix it :)

I considered your remarks and pushed the modifications.

**Comment by mgechev** [comment]
LGTM

**Comment by mgechev** [comment]
@ojacquemart would you rebase against master?

**Comment by ojacquemart** [comment]
@mgechev It should be ok

**Pull request merged**